### ISSUE TITLE
Cross-platform replacement for SoundPlayer for playing audio across devices/paltforms

### ISSUE THREAD

**Issue by Petermarcu** [dotnet/corefx#32413]
@jjgriff93 commented on [Thu Sep 20 2018](https://github.com/dotnet/core/issues/1952)

Cannot play sound files as part of .net core console apps in windows and running on windows IoT Core on Raspberry Pi due to lack of audio support. Third party libraries don't offer any simple solution for playing wav or mp3 streams like the .NET Framework SoundPlayer does.

**Comment by Petermarcu** [comment]
@richlander

**Comment by karelz** [comment]
@jjgriff93 what are the scenarios where it is needed? What kind of apps would benefit from the APIs?
I assume the API is [System.Media.SoundPlayer](https://docs.microsoft.com/en-us/dotnet/api/system.media.soundplayer?view=netframework-4.7.2)

**Comment by synercoder** [comment]
@karelz I wish to create a "digital assistent" in a windows console app (perhaps later hosted on a raspberry pi). That uses Azure services for text-to-speech for the responses. Currently I can't do this in netcore.
So that is one scenario example.

**Comment by keithn** [comment]
is this going to be in .net core 3?

**Comment by karelz** [comment]
No, it is in Future milestone. Also note that there is fairly small interest from community (4 people so far).

**Comment by mdisg** [comment]
Would be very helpfull to get something like the SoundPlayer provides for windows but universal for other platforms like linux using alsa. 
We use .NET Core on a embedded device which regulary has to play some short sounds. Bringing up a command line utlity like aplay is not that good in performance.
